go/gazelle/BUILD

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+load("@bazel_gazelle//:def.bzl", "gazelle")
+
+# for updating dependencies
+gazelle(
+    name = "gazelle-update-dependencies",
```

Comment:
I'd rename it to `update-dependencies`, because it's a target of the `gazelle` package. Your `.sh` script shows how a bit funny it looks otherwise: `//go/gazelle:gazelle-update-dependencies`